# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1511957391 -3600
#      Wed Nov 29 13:09:51 2017 +0100
# Node ID 24a6452059fd3eb6e91e5c24a53710e09fe7451b
# Parent  f38165f34ff1a01afcb9adf624abeb2a3813bdfc
doc: Add missing contributors from hg log to contributors.in (bug #46740)

* doc/interpreter/contributors.in: add missing contributors from hg log.  The
method is explained in bug #46740.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -93,16 +93,17 @@ Edmund Grimley Evans
 Rolf Fabian
 Francesco Faccio
 Gunnar Farnebäck
 Massimiliano Fasi
 Stephen Fegan
 Ramon Garcia Fernandez
 Torsten Finke
 David Finkel
+Guillaume Flandin
 Colin Foster
 Jose Daniel Munoz Frias
 Brad Froehle
 Castor Fu
 Eduardo Gallestey
 Walter Gautschi
 Klaus Gebhardt
 Driss Ghaddab
@@ -177,16 +178,17 @@ Thomas Kasper
 Joel Keay
 Mumit Khan
 Paul Kienzle
 Lars Kindermann
 Aaron A. King
 Erik Kjellson
 Arno J. Klaassen
 Alexander Klein
+Lasse Kliemann
 Geoffrey Knauth
 Heine Kolltveit
 Ken Kouno
 Kacper Kowalik
 Endre Kozma
 Daniel Kraft
 Nir Krakauer
 Aravindh Krishnamoorthy
@@ -202,16 +204,17 @@ Miroslaw Kwasniak
 Rafael Laboissiere
 Kai Labusch
 Claude Lacoursiere
 Walter Landry
 Bill Lash
 Dirk Laurie
 Maurice LeBrun
 Friedrich Leisch
+Michael Leitner
 Johannes Leuschner
 Thorsten Liebig
 Torsten Lilge
 Jyh-miin Lin
 Timo Lindfors
 Benjamin Lindner
 Ross Lippert
 Yu Liu
@@ -242,16 +245,17 @@ Júlio Hoffimann Mendes
 Ed Meyer
 Thorsten Meyer
 Stefan Miereis
 Petr Mikulik
 Mike Miller
 Serviscope Minor
 Stefan Monnier
 Rafael Monteiro
+Stephen Montgomery-Smith
 Antoine Moreau
 Kai P. Mueller
 Amod Mulay
 Armin Müller
 Hannes Müller
 Victor Munoz
 PrasannaKumar Muralidharan
 Iain Murray
@@ -262,16 +266,17 @@ Todd Neal
 Philip Nienhuis
 Al Niessner
 Felipe G. Nievinski
 Rick Niles
 Takuji Nishimura
 Akira Noda
 Kai Noda
 Patrick Noffke
+Victor Norton
 Eric Norum
 Krzesimir Nowak
 Michael O'Brien
 Cillian O'Driscoll
 Peter O'Gorman
 Thorsten Ohl
 Kai T. Ohlhus
 Arno Onken
@@ -297,16 +302,17 @@ Sergey Plotnikov
 Tom Poage
 Nathan Podlich
 Orion Poplawski
 Ondrej Popp
 Jef Poskanzer
 Francesco Potortì
 Konstantinos Poulios
 Tejaswi D. Prakash
+Hugo Raguet
 Jarno Rajahalme
 Eduardo Ramos
 Pooja Rao
 James B. Rawlings
 Eric S. Raymond
 Balint Reczey
 Joshua Redstone
 Andy Register
@@ -360,16 +366,17 @@ John Smith
 Julius Smith
 Shan G. Smith
 Peter L. Sondergaard
 Rüdiger Sonderfeld
 Joerg Specht
 Quentin H. Spencer
 Christoph Spiel
 David Spies
+Andreas Stahel
 Richard Stallman
 Russell Standish
 Ryan Starret
 Brett Stewart
 Doug Stewart
 Jen Stewart
 Jonathan Stickel
 Judd Storrs
@@ -411,18 +418,20 @@ Olaf Weber
 Thomas Weber
 Rik Wehbring
 Bob Weigel
 Andreas Weingessel
 Martin Weiser
 Michael Weitzel
 David Wells
 Joachim Wiesemann
+Alexander Wilms
 Georg Wiora
 Fook Fah Yap
 Sean Young
 Michele Zaffalon
 Serhiy Zahoriya
 Johannes Zarl
 Michael Zeising
 Federico Zenith
 Claudius Zingerli
 Alex Zvoleff
+Richard Zweig
